definitions/variable/emissions/carbon-management.yaml

```
@@ -0,0 +1,97 @@
+# Carbon Capture is grouped into two levels of aggregation based on the fate of the captured CO2.
+# Level-1 reports whether captured CO2 is stored, utilized, or leaked
+# Level-2 reports additional disaggregation for long-lived vs. short-lived utilization.
+
+- Carbon Capture:
+    description: Captured carbon dioxide (CO2) from energy use in supply and demand sectors (IPCC category 1A, 1B) and from industrial processes 
+        (IPCC categories 2A, B, C, E), including emissions from international bunkers, net of negative emissions from bioenergy with CCS (BECCS), 
+        direct air carbon capture and storage (DACCS), and other CO2 removal (exclusive of land-use change) in these sectors
+    unit: Mt CO2/yr
+
+- Carbon Capture|Geologic Storage:
+    description: Captured carbon dioxide (CO2) that is injected into subsurface geologic storage reservoirs (e.g., in situ mineralization).
+    unit: Mt CO2/yr
+    note: Captured CO2 that is reacted at the earth's surface (e.g., ex situ mineralization) should be reported under Carbon Capture|Other Storage
+
+- Carbon Capture|Leakage:
+    description: Captured carbon dioxide (CO2) that is subsequently vented, leaked, or reemitted during or after transmission to storage repositories
+    unit: Mt CO2/yr
+    note: Leakage of CO2 stored in previous years should be reported here
+    
+-Carbon Capture|Other Storage:
```

The missing whitespace is causing the tests to fail.
```suggestion
- Carbon Capture|Other Storage:
```